### PR TITLE
Use SPDX license headers

### PR DESCRIPTION
**Pull request by pcercuei**
Use simple SPDX license strings instead of having paragraphs of text about the license in the header of every single source file of the project.

Also I'm pretty sure @cristi-iacob's first name is not Paul.

### CODE REVIEW

File: bindings/csharp/IioLib.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * libiio - Library for interfacing industrial I/O (IIO) devices
  *
  * Copyright (C) 2020 Analog Devices, Inc.
- * Author: Paul Cristian Iacob <cristian.iacob@analog.com>
+ * Author: Cristian Iacob <cristian.iacob@analog.com>
```
Comment (mirceacaprioru) [inline bindings/csharp/IioLib.cs:6]: actually his name really is Paul Cristian Iacob.

Comment (pcercuei) [reply bindings/csharp/IioLib.cs:6]: Ha :D
All other headers just had "Cristian Iacob", so I thought it was a bad copy-paste.

**Comment by mirceacaprioru** [comment]
just one nitpick some places have 
```
/* SPDX-License-Identifier: LGPL-2.1-or-later */
```
while others
`// SPDX-License-Identifier: LGPL-2.1-or-later`

But this is not a strong request. Just an observation

**Review by mirceacaprioru — APPROVED** [review]

**Comment by pcercuei** [comment]
> just one nitpick some places have
> 
> ```
> /* SPDX-License-Identifier: LGPL-2.1-or-later */
> ```
> 
> while others
> `// SPDX-License-Identifier: LGPL-2.1-or-later`
> 
> But this is not a strong request. Just an observation

Yes, it does look strange, but this is the convention - /* */ in headers and // in C source files.

**Pull request merged**